libs/api.lua

```diff
@@ -136,29 +200,32 @@ return function (prefix)
       return next(authors) and authors
     end,
     "^/search/(.*)$", function (query)
+      -- escape all special pattern characters except . and *
+      query = query:gsub("[%(%)%%%+%-%?%[%]%^%$]", function(c) return "%" .. c end)
       local matches = {}
+      local authorQuery, separator, packageQuery = query:match("^(.-)(/?)([^/]*)$")
```

Comment:
Right, that convention follows how `lit-browser.js` would separate it (`author/author/../package`), which is what I used as a reference.